zkvm/src/types.rs

```diff
@@ -115,6 +116,26 @@ impl CommitmentWitness {
         let gens = PedersenGens::default();
         gens.commit(self.value.into(), self.blinding).compress()
     }
+
+    pub fn unblinded<T>(x: T) -> Self
+    where
+        T: Into<ScalarWitness>,
+    {
+        CommitmentWitness {
+            blinding: Scalar::zero(),
+            value: x.into(),
+        }
+    }
+
+    pub fn from_secret<T>(x: T) -> Self
```

Comment:
maybe `blinded` for symmetry with `unblinded`?

Reply:
👍 I was thinking that too!